libr/util/format.c

```diff
@@ -2359,7 +2373,15 @@ R_API int r_print_format(RPrint *p, ut64 seek, const ut8* b, const int len,
 					break;
 				case 'q':
 					r_print_format_quadword (p, endian, mode, setval, seeki, buf, i, size);
-					i += (size==-1) ? 8 : 8*size;
+					i += (size == -1) ? 8 : 8  * size;
```

Comment:
wrong `X? Y: Z` spaces and also `8  * size`. Since you are already adjusting this line..